module.json

```diff
@@ -37,6 +37,12 @@
             "name": "English",
             "path": "lang/en.json",
             "flags": {}
+        },
+        {
+            "lang": "cs",
+            "name": "Česky",
+            "path": "lang/cs.json"
```

Comment:
Each line of an object in a json will need a trailing comma or the module will be unable to load.

(Unless it is the last line in the object.)

Reply:
Fixxed and found parts that cannot be translated -> https://github.com/krbz999/visual-active-effects/issues/7